### ISSUE TITLE
[MC 1.15.2] CraftTweaker doesn't remove recipes from this mod

### ISSUE THREAD

**Issue by mosharky** [Flanks255/simplybackpacks#16]
**Describe the bug**
CraftTweaker; latest version, doesn't remove recipes from this mod. I've tried the exact same code on other mods.

**Screenshots**
These are the zenscript lines
![image](https://user-images.githubusercontent.com/32606683/78548874-739fa780-77cf-11ea-88db-6fcedc4af5e9.png)

Here's what the Rare backpack looks like in-game, when it's supposed to remove the original recipe.
![image](https://user-images.githubusercontent.com/32606683/78548974-9fbb2880-77cf-11ea-8a01-e577c187655d.png)

Here's the common backpack, able to display the information line I gave in the last line of code, but not remove its recipe.
![image](https://user-images.githubusercontent.com/32606683/78549050-beb9ba80-77cf-11ea-8ac1-61e19f692023.png)
![image](https://user-images.githubusercontent.com/32606683/78549063-c2e5d800-77cf-11ea-9442-edb5ec73d3ba.png)

CraftTweaker version 6.0.0.10
Mod version version 1.4.5
Forge version version 31.1.34

**Comment by Flanks255** [comment]
so, removeByName, takes a recipe name, not an item name.
you can see recipe names by turning on advanced tooltips with F3+H, and hovering over the output item in JEI while viewing a recipe.

so change
`craftingTable.removeByName("simplybackpacks:commonbackpack");
craftingTable.removeByName("simplybackpacks:uncommonbackpack");
craftingTable.removeByName("simplybackpacks:rarebackpack");
craftingTable.removeByName("simplybackpacks:epicbackpack");
`
to
`craftingTable.removeByName("simplybackpacks:common");
craftingTable.removeByName("simplybackpacks:uncommon");
craftingTable.removeByName("simplybackpacks:rare");
craftingTable.removeByName("simplybackpacks:epic");
`

TL;DR Item Name != Recipe Name

**Comment by mosharky** [comment]
Thanks!

**Comment by Flanks255** [comment]
It shows up when viewing the recipe in JEI, not in inventory.
